content/doc/book/pipeline/syntax.adoc

```diff
@@ -1231,14 +1231,13 @@ node {
 === Steps
 
 As discussed in <<getting-started#, Getting Started>>, the most fundamental
-part of a Pipeline is the "step." Fundamentally, steps tell Jenkins _what_ to
+part of a Pipeline is the "step". Fundamentally, steps tell Jenkins _what_ to
```

Comment:
Is this not correct as is?

Reply:
I believe that's correct - afaik, the period should only occur before the final quotation mark if the entire sentence is enclosed in quotation marks:
https://forum.wordreference.com/threads/full-stop-inside-or-outside-quote.2459638/

Reply:
Hmm - I should've read the entire thread before blurting out. After reading through this article (skimming more like it), I couldn't find a specific authority on this, but it appears like there are differences between US and UK/Aussie English (which I'm used to) in the way punctuation is handled with respect to quotation final marks...

In US English, punctuation marks are written before the final quotation mark - regardless of what the quotation mark is encapsulating.

In UK/Aussie English, punctuation marks are only written before the final quotation mark if they are part of the speech which is being quoted, or the entire sentence is quoted. In all other cases, the punctuation marks occurs outside/after the final quotation mark.

Reply:
Thanks for sharing the link @gilesgas 

I was just thinking the [same](https://forum.wordreference.com/threads/full-stop-inside-or-outside-quote.2459638/#post-12376918) :)